src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/SinkIslandCheck.java

```diff
@@ -183,30 +189,28 @@ public boolean validCheckForObject(final AtlasObject object)
     private boolean validEdge(final AtlasObject object)
     {
         return object instanceof Edge
-                // Ignore any airport taxiways and runways, as these often create a sink island
-                && !Validators.isOfType(object, AerowayTag.class, AerowayTag.TAXIWAY,
-                        AerowayTag.RUNWAY)
                 // Only allow car navigable highways and ignore ferries
-                && HighwayTag.isCarNavigableHighway(object) && !RouteTag.isFerry(object)
+                && this.isCarNavigableHighway((Edge) object) && !RouteTag.isFerry(object)
                 // Ignore any highways tagged as areas
                 && !TagPredicates.IS_AREA.test(object);
     }
 
     /**
      * This function checks to see if the end node of an Edge AtlasObject has an amenity tag with
-     * one of the AMENITY_VALUES_TO_EXCLUDE.
+     * one of the AMENITY_VALUES_TO_EXCLUDE or has a start node with amenity value as
+     * "parking_entrance"
      * 
      * @param object
      *            An AtlasObject (known to be an Edge)
-     * @return {@code true} if the end node of the end has one of the AMENITY_VALUES_TO_EXCLUDE, and
-     *         {@code false} otherwise
+     * @return {@code true} if the end node of the end has one of the AMENITY_VALUES_TO_EXCLUDE or
+     *         start node has "amenity=parking_entrance", and {@code false} otherwise
      */
-    private boolean endNodeHasAmenityTypeToExclude(final AtlasObject object)
+    private boolean endOrStartNodeHasAmenityTypeToExclude(final AtlasObject object)
     {
         final Edge edge = (Edge) object;
         final Node endNode = edge.end();
-
-        return Validators.isOfType(endNode, AmenityTag.class, AMENITY_VALUES_TO_EXCLUDE);
+        return Validators.isOfType(endNode, AmenityTag.class, AMENITY_VALUES_TO_EXCLUDE)
+                || Validators.isOfType(edge.start(), AmenityTag.class, AmenityTag.PARKING_ENTRANCE);
```

Comment:
nit: could we replace `endNode` with `edge.end()` here? Or add `final Node startNode = edge.start()`? Just make the two lines more similar for readability